src/coreclr/md/ceefilegen/cceegen.cpp

```diff
@@ -399,9 +400,10 @@ HRESULT CCeeGen::emitMetaData(IMetaDataEmit *emitter, CeeSection* section, DWORD
 {
     HRESULT hr = S_OK;
 
-    ReleaseHolder<IStream> metaStream(NULL);
+    ReleaseHolder<IStream> metaStream(new(std::nothrow) CGrowableStream());
```

Comment:
Are we sure this types doesn't do an `AddRef()` via the constructor? The inheritance heirarchy here is a mess for no obvious reason.

Reply:
This type doesn't do any sort of ref tracking on acquisition. It only releases on destruction.